### DISCUSSION TITLE
some problem with the mex file

### DISCUSSION THREAD

**Discussion by darkblue3699** [e0404/matRad#644] [Q&A]
![image](https://github.com/e0404/matRad/assets/42909701/1bf18875-64f7-40d6-b751-d650cb253a43)
I run the script 'matRad_example1_phantom.m'. It shows the error that tell me the optimize mex file is invalid. But  I have the ipopt.mexw64 file in the directory optimization/optimizer.

**Comment by wahln** [comment]
Which Matlab version are you using? the current ipopt.mexw64 may be incompatible to older Matlab versions (<2019). 
If you have the optimization toolbox installed, you can also try to set `pln.propOpt.optimizer = 'fmincon';` to optimize with the interior-point method implemented by Mathworks.

**Reply by darkblue3699** [reply]
Thanks for your reply. My matlab version is 2016. I will change to new version Matlab.
